### PR TITLE
Add the option that shows the mini cart into a sidebar or a pop-up

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
Add the option that shows the mini cart into a sidebar or a pop-up.

#### What problem is this solving?
Enable to choose if the mini cart will be shown into a Pop-up ou into a Sidebar.

#### How should this be manually tested?
https://waza--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by brunojdo — APPROVED** [review]

File: react/components/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import { IconCaretRight } from 'vtex.styleguide'
+import { injectIntl, intlShape } from 'react-intl'
+
+import OutsideClickHandler from 'react-outside-click-handler'
+
+import MiniCart from '../MiniCart'
+
+const whiteColor = '#FFFFFF'
```
Comment (salesfelipe) [inline react/components/Sidebar.js]: genererally constants are written in upper case. WHITE_COLOR

File: react/components/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  componentWillUnmount() {
+    document.body.classList.remove('vtex-minicart-sidebar-open')
+  }
+
+  render() {
+    const { onBackClick, intl } = this.props
+
+    if (typeof window !== 'undefined') {
```
Comment (salesfelipe) [inline react/components/Sidebar.js:26]: Is this really necessary, manipulating the window and the body?

Comment (vitoria) [reply react/components/Sidebar.js:26]: Isso é necessário para deixar a sidebar na altura 100% da tela.

Comment (salesfelipe) [reply react/components/Sidebar.js:26]: if you say so...

**Review by salesfelipe — APPROVED** [review]

**Pull request merged**